test/test_quantize_script.py

```diff
@@ -71,5 +72,101 @@ def forward(self, x):
                    .check_not('Observer = prim::GetAttr[name="_observer_') \
                    .run(m.graph)
 
+
+    def test_insert_quant_dequant_dynamic(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 5, 3).float()
+
+            def forward(self, x):
+                return self.conv(x)
+
+        m = torch.jit.script(M())
+        qconfig = default_dynamic_qconfig
+        qconfig_dict = {
+            '': script_qconfig(qconfig)
+        }
+        m = wrap_cpp_module(torch._C._jit_pass_insert_observers(m._c, "forward", qconfig_dict, False, True))
```

Comment:
could you use prepare_dynamic_script here? also I added support for None in qconfig_dict, could you make this a function: https://github.com/pytorch/pytorch/blob/master/torch/quantization/_quantize_script.py#L76 and call it in prepare_dynamic_script as well?